site/content/docs/main/contributions/ibm-config.md

```diff
@@ -75,12 +76,6 @@ Additionally, you can specify `--use-node-agent` to enable [File System Backup][
 
 (Optional) Specify [CPU and memory resource requests and limits][15] for the Velero/node-agent pods.
 
-Once the installation is complete, remove the default `VolumeSnapshotLocation` that was created by `velero install`, since it's specific to AWS and won't work for IBM Cloud:
```

Comment:
It did not appear to happen when I installed using the instructions mentioned...but I'm happy to restore those lines if you believe it might be required.
I'm using a local IBM cloud storage (s3 compatible)

Reply:
OK. Please restore those lines.
The rest part of this PR looks good.